wiremock-common/src/main/java/com/github/tomakehurst/wiremock/recording/SnapshotOutputFormatter.java

```diff
@@ -20,21 +20,24 @@
 import java.util.List;
 
 /** Wraps a list of generated StubMappings into a SnapshotRecordResult object */
-public enum SnapshotOutputFormatter {
+enum SnapshotOutputFormatter {
   FULL {
     @Override
-    public SnapshotRecordResult format(List<StubMapping> stubMappings) {
-      return SnapshotRecordResult.full(stubMappings);
+    public SnapshotRecordResult format(
+        List<StubMapping> stubMappings, List<? extends RecordError> errors) {
+      return SnapshotRecordResult.full(stubMappings, errors);
     }
   },
   IDS {
     @Override
-    public SnapshotRecordResult format(List<StubMapping> stubMappings) {
-      return SnapshotRecordResult.idsFromMappings(stubMappings);
+    public SnapshotRecordResult format(
+        List<StubMapping> stubMappings, List<? extends RecordError> errors) {
+      return SnapshotRecordResult.idsFromMappings(stubMappings, errors);
     }
   };
 
-  public abstract SnapshotRecordResult format(List<StubMapping> stubMappings);
+  public abstract SnapshotRecordResult format(
```

Comment:
Can this method (and its overrides) also be package private?

Reply:
I think the methods can be package private, but the class needs to revert to being public because it is exposed as public properties of `RecordSpec` and `RecordSpecBuilder`